File: apay_android/src/main/java/kz/airbapay/apay_android/ui/pages/card_reader/bl/OnScanListener.java
package kz.airbapay.apay_android.ui.pages.card_reader.bl;

import android.graphics.Bitmap;

import java.util.List;

interface OnScanListener {

	void onPrediction(final String number, final Bitmap bitmap, final List<DetectedBox> digitBoxes);
	void onFatalError();

}

File: apay_android/src/main/java/kz/airbapay/apay_android/ui/pages/card_reader/bl/camera/OnScanListener.kt
package kz.airbapay.apay_android.ui.pages.card_reader.bl.camera

import android.graphics.Bitmap
import kz.airbapay.apay_android.ui.pages.card_reader.bl.card_number_detection.DetectedBox

internal interface OnScanListener {
    fun onPrediction(number: String?, bitmap: Bitmap?, digitBoxes: List<DetectedBox?>?)
    fun onFatalError()
}